### ISSUE TITLE
MacOS Arm64 0.9.2 Binary - "GemRB" is damaged and can't be opened. You should move it to the Trash. 

### ISSUE THREAD

**Issue by davidagnome** [gemrb/gemrb#1953]
#### Bug description
I've downloaded the ZIP files from Sourceforge twice now. I unzipped with two different programs in case it was an issue with the Unarchiver. Launching GemRB results 

#### Steps to reproduce
1. Download MacOS Arm64 0.9.2 ZIP from SourceForge
2. Extract file
3. Open GemRB.

#### Expected behavior
Program should launch.

#### Screenshots
![Screenshot 2023-12-02 at 9 32 29 AM](https://github.com/gemrb/gemrb/assets/5984719/0699dc85-148e-4305-b46c-b836f68c58c0)

GemRB version (check as many as you know apply)
- [ ] master as of this issue
- [X] 0.9.2
- [ ] 0.9.1
- [ ] 0.9.0

Video Driver (check as many as you know apply)
- [ ] SDL1.2
- [ ] SDL2 built with `USE_OPENGL_BACKEND`
- [ ] SDL2 without `USE_OPENGL_BACKEND`

**Comment by lynxlynxlynx** [comment]
It's probably built for a different macos version, that's why (running it in the terminal would give you an actual error to think about). You can't do anything but build it yourself until #1861 and #1863 get fixed.

**Comment by lynxlynxlynx** [comment]
No, I'm pretty sure it's the same brew problem as discussed elsewhere, where the brew target version doesn't match.

**Comment by lynxlynxlynx** [comment]
It's done by default in newer macoses for downloads; we can't pre-set this attribute unfortunately. This _is_ something that signing could fix though.

It looks easier now, definitely feasible for releases:
https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution/customizing_the_notarization_workflow#3087734
The nondeterministic timing makes it less than ideal for buildbot builds, but anyone using them surely knows how to work around this, so we could just ignore them.
